CONTRIBUTING.md

```diff
@@ -33,7 +33,7 @@ The goal of the community is to develop a cloud native edge computing platform b
 # Getting started
 
 - Fork the repository on GitHub
-- Read the [setup](https://kubeedge.io/en/docs/setup/keadm/) for deployment.
+- Read the [setup](https://kubeedge.io/docs/setup/install-with-keadm) for deployment.
```

Comment:
`- Read the [Developer Guide][https://kubeedge.io/docs/category/developer-guide]`
Would it be better to use the Developer Guide here? @fisherxu

Reply:
Good idea! I think we can have all of them, how about add `[Developer Guide]` below `[setup]`? @VamshiReddy02 @WillardHu

Reply:
SGTM

Reply:
working on it